src/ast/schemaPredicates.test.ts

```diff
@@ -29,4 +29,13 @@ describe('SchemaPredicates', () => {
     expect(schemaPredicates.isFieldNullable('Todo', 'complete')).toBeTruthy();
     expect(schemaPredicates.isFieldNullable('Todo', 'author')).toBeTruthy();
   });
+
+  it('should handle unions of objects', () => {
+    expect(
+      schemaPredicates.isInterfaceOfType('LatestTodoResult', 'Todo')
```

Comment:
So, now it makes a little bit more sense to me; this would be the result of

```graphql
type Todo {

}

type NoTodosError {

}

union LatestTodoResult = Todo | NoTodosError
```

Reply:
I suppose we may want to rename this method now to sth like `isMatchingCondition`

Reply:
I'll refrain from commenting further on what does and doesn't make sense until I've spent some more time on GraphQL primitives. The fact that it was counterintuitive to me probably has more to do with that than the actual API.

Reply:
To be honest, I can't really reason about how this test actually implies correctness. What's the failure path here?